.prettierrc.json

```diff
@@ -2,5 +2,6 @@
 	"printWidth": 100,
 	"singleQuote": true,
 	"singleAttributePerLine": true,
-	"tabWidth": 4
+	"tabWidth": 4,
+	"trailingComma": "es5"
```

Comment:
Why?

Reply:
when I was saving, it kept removing trailing commas, then giving me a warning telling me that it wanted the trailing commas